plugins/frontend/backstage-plugin-github-pull-requests/.changeset/sixty-apples-cough.md

```diff
@@ -0,0 +1,5 @@
+---
+'@roadiehq/backstage-plugin-github-pull-requests': minor
```

Comment:
This looks to be a patch change since it doesn't introduce new functionality or breaking changes.